samples/snippets/src/test/java/com/example/spanner/SampleRunner.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.concurrent.Callable;
+
+public class SampleRunner {
+  public static String runSample(Callable<Void> sample) throws Exception {
+    final PrintStream stdOut = System.out;
+    final ByteArrayOutputStream bout = new ByteArrayOutputStream();
+    final PrintStream out = new PrintStream(bout);
+    System.setOut(out);
+    sample.call();
+    System.setOut(stdOut);
+    return bout.toString();
+  }
+
+  public interface Sample {
```

Comment:
AFAICT, this interface is not used anywhere

Reply:
Removed.